pkgs/applications/networking/browsers/chromium/update.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh -e
 cd "$(dirname "$0")"
-sp="$(nix-build -Q --no-out-link update.nix -A update)"
+sp="$(nix-build -Q --no-out-link update.nix -A update --option build-use-sandbox false)"
```

Comment:
I assume update.sh is run manually, not through the nix daemon no? Anyway, I see that update.nix already fetches the tarballs without any hashes, so I assume this is already run outside a sandbox.